src/sktool/tests/n2k-transducer.json

```diff
@@ -19,6 +19,8 @@
       { "path": "environment.depth.belowTransducer", "value": 3.45 },
       { "path": "environment.depth.belowSurface", "value": 4.364 },
       { "path": "environment.depth.surfaceToTransducer", "value": 0.914 }
+      { "path": "environment.depth.transducerToKeel", "value": -1000000000.0 }
```

Comment:
No. I don't think this is correct. We should not generate these values for this incoming message. 

The SignalK tools do not do it, and I would bet that iKommunicate does not do it either.

Reply:
May be, but they could be wrong... :joy:

There was an error in the test, which was gone after I entered this....

Reply:
Took it out again

Reply:
😧 There is absolutely no way that `-1000000000.0` is correct here :no_good: 

Either we generate: `"value": null` or we do not generate an entry for `transducerToKeel` at all. In general SignalK omits values that are `null` or `undefined`. This is what we should do here. I believe it is correct.

> There was an error in the test, which was gone after I entered this....

or maybe the test was failing because the code is incorrect 😜